projects/js-packages/publicize-components/src/components/settings-button/index.js

```diff
@@ -34,20 +40,22 @@ export default function PublicizeSettingsButton() {
 		refresh();
 	}, refreshThreshold );
 
+	const connectionsUrl =
+		getJetpackData()?.publicizeConnectionsUrl ?? 'https://wordpress.com/marketing/connections/';
```

Comment:
This is a temporary measure. We're not ready to send current Jetpack sites to Jetpack Cloud for the connections screen, so for now we'll default to Calypso, and this lets us change that with the initial state data.